src/arithmetic/func_desc.c

```
@@ -4,21 +4,100 @@
+} UDFS ;
+
+UDFS *__udfs              = NULL ;
+rax  *__aeRegisteredFuncs = NULL ;
+
+static void _NormalizeFunctionName
+(
+	char *name,
+	char normalized[FUNC_NAME_MAX_LEN],
+	size_t *len
+) {
+	ASSERT (len  != NULL) ;
```

Cppcheck (2.18.0)

[error] Null pointer dereference

(nullPointer)